### PR TITLE
Refactored Payments to Online Payments

### PR DESCRIPTION
**Pull request by MW-Forster**

### CODE REVIEW

File: src/EPR.Payment.Mopup.Common.Data/Repositories/PaymentsRepository.cs [hunk 1]
```diff
@@ -32,14 +32,12 @@ public async Task UpdatePaymentStatusAsync(DataModels.Payment? entity, Cancellat
...
-        public async Task<List<DataModels.Payment>> GetPaymentsByStatusAsync(Status status, CancellationToken cancellationToken)
+        public async Task<List<DataModels.OnlinePayment>> GetPaymentsByStatusAsync(Status status, CancellationToken cancellationToken)
         {
             DateTime now = DateTime.UtcNow;
             DateTime updateFrom = now.AddMinutes(-Convert.ToInt32(_configuration["TotalMinutesToUpdate"]));
             DateTime ignoringFrom = now.AddMinutes(-Convert.ToInt32(_configuration["IgnoringMinutesToUpdate"]));
-            var entities = await _dataContext.Payment
-                .Where(a => a.InternalStatusId == Status.InProgress && a.CreatedDate >= updateFrom && a.CreatedDate <= ignoringFrom)
-                .ToListAsync(cancellationToken);
+            var entities = await _dataContext.Payment.Where(a => a.InternalStatusId == Status.InProgress && a.CreatedDate >= updateFrom && a.CreatedDate <= ignoringFrom).ToListAsync();
```
Comment (davidjayakumarEviden) [inline src/EPR.Payment.Mopup.Common.Data/Repositories/PaymentsRepository.cs:40]: cancellationToken is not passed as parameter ToListAsync(). Also retain the formatting into multiple lines

**Review by davidjayakumarEviden — COMMENTED** [review]
Just one comment

File: src/EPR.Payment.Mopup.Common.Data/Repositories/PaymentsRepository.cs (hunk 1 above)
Comment (MW-Forster) [reply src/EPR.Payment.Mopup.Common.Data/Repositories/PaymentsRepository.cs]: Added Token and updated formatting

**Review by davidjayakumarEviden — APPROVED** [review]
Looks good

**Pull request closed without merging**